README.md

```diff
@@ -69,6 +69,7 @@ Supported Cars
 | Acura     | RDX 2020                      | AcuraWatch        | Stock            | 0mph               | 3mph              |
 | Honda     | Accord 2018-20                | All               | Stock            | 0mph               | 3mph              |
 | Honda     | Accord Hybrid 2018-20         | All               | Stock            | 0mph               | 3mph              |
+| Honda     | Inspire 2018                  | All               | Stock            | 0mph               | 3mph              |
```

Comment:
let's remove this since the README is supposed to be a US market reference, but let's add a comment in values.py